datasets/management/commands/crawl.py

```diff
@@ -62,6 +67,10 @@ def handle(self, *args, **options):
         os.environ["SCRAPY_SETTINGS_MODULE"] = "scraper.settings"
         process = CrawlerProcess(settings=get_project_settings())
         process.crawl(AgendaSpider)
+        process.crawl(
+            AttendanceListSpider,
+            start_from_date=CityCouncilAttendanceList.last_collected_item_date(),
```

Comment:
Os status podem ser atualizados ao longo do mês... me pergunto se, pra esse caso, não deveríamos coletar do início do ano ou do início do mês e atualizar as presenças sempre. :thinking: